catalog/plugin/catalog-plugin-auditcontrolledaccess/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+ -->
+<metatype:MetaData xmlns:metatype="http://www.osgi.org/xmlns/metatype/v1.0.0">
+
+  <OCD description="Audit Controlled Access Plugin"
+    name="Audit Controlled Access Plugin"
+    id="org.codice.alliance.catalog.plugin.auditcontrolled.AuditControlledAccessPlugin">
+
+    <AD name="Classification:" id="controlledClassificationValues"
+      description="Classification values to audit."
+      required="false" type="String" cardinality="100" default=""/>
+
+    <AD name="Releasability:" id="controlledReleasabilityValues"
+      description="Releasability values to audit."
+      required="false" type="String" cardinality="100" default=""/>
+
+    <AD name="DisseminationControls:" id="controlledDisseminationControlsValues"
+      description="DisseminationControls values to audit."
+      required="false" type="String" cardinality="100" default=""/>
+
+    <AD name="Codewords:" id="controlledCodewordsValues" description="Codewords values to audit."
+      required="false" type="String" cardinality="100" default=""/>
+  </OCD>
```

Comment:
❓ I see the default value for these metatypes is blank. Does that mean, lets say Classification is blank, that *any* classification will be audited or *no* classifications will be audited? Might also be a good idea to add that answer to the description as well.

Reply:
_No_ values with be audited by default. I have seen other references to plugins where the values are defaulted to blank, which means nothing will occur. The docs explain that `If any of the configured values match the respective attribute value(s) on the metacard, the metacard is audited.` To me, it seems a bit overkill to explain nothing will be audited by default, but I could see why others might want extra clarification. Perhaps @ricklarsen could review and respond to this comment?